glossary.md

```diff
@@ -255,6 +255,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | max pooling                  | gộp cực đại                   | [https://git.io/JfGi6](https://git.io/JfGi6) |
 | maximum likelihood estimator | bộ ước lượng hợp lý cực đại   |                                              |
 | mean squared error (MSE)     | trung bình bình phương sai số | [https://git.io/Jvojr](https://git.io/Jvojr) |
+| memory cell                  | đơn vị bộ nhớ                 |                                              |
```

Comment:
hình như trong tin học chung chung họ dịch thế này
```suggestion
| memory cell                  | ô nhớ                 |                                              |
```

Reply:
Mình nghĩ ngữ cảnh ở tin học chung chung có lẽ khác ở đây.

Reply:
Theo gợi ý của Đức, mình đang phân vân `đơn vị nhớ` hợp lý hơn. `LSTM cell` có lẽ nên dịch là 1 `đơn vị LSTM`?

Reply:
@davidnvq yeah mình thấy ổn

Reply:
```suggestion
| memory cell                  | khối nhớ                 |                                              |
```
và `khối LSTM` thì sao nhỉ? Nó sẽ cùng nghĩa với từ `Block` trong lúc lập trình.

Reply:
Nếu thế chắc ngày xưa họ đã dùng từ `block` rồi :-? Mà memory cell ở đây nó lại còn là 1 vector đơn thuần thôi. Ông tổ LSTM nghĩ ra nhiều từ lạ ghê :)))

Reply:
`ngăn nhớ` nghe ổn ko nhỉ?
```suggestion
| memory cell                  | ngăn nhớ                 |                                              |
```

Reply:
`Block` ở các chương trước (CNN), thì ngầm quy ước là 1 khối gồm nhiều layers, còn `LSTM cell` là 1 phần trong LSTM layer. Dù `block` đặc biệt là `building block` thì k nhất thiết phải là 1 khối nhiều layers như vậy.

Reply:
Rồi sang bên `LSTM cell` thì dịch là ngăn LSTM có vẻ k xuôi

Reply:
ok em đồng ý `đơn vị nhớ`.

Reply:
> còn `LSTM cell` là 1 phần trong LSTM layer.

Có gì đó không đúng. Mình hiểu là trong `LSTM layer/cell/unit/block` có một `memory cell` và 3 cổng, không biết còn thiếu gì không?

Mình cũng vừa search qua toàn bộ chương 9 không thấy có chỗ nào xài từ `LSTM cell`, và trong các cách dùng phổ biến cũng ít xài `LSTM cell` nên mình tạm thời không cần quan tâm cách dịch từ này.

 Trong trường hợp đó thì mình ủng hộ `ô nhớ` hay `ngăn nhớ` như đề xuất ở trên.

Reply:
Ủa không có LSTM Cell à, chắc mình nhớ lộn là từ trong cuốn D2L. Btw, từ LSTM Cell lại xuất hiện khá thường xuyên trong Pytorch, Gluon, Chainer và Tensorflow.

Reply:
Yeah, mà nó đều là Layer / Block cả nhỉ? https://stackoverflow.com/questions/57048120/pytorch-lstm-vs-lstmcell
Chữ `cell` này mình thấy không có nghĩa gì đặc biệt nên cũng không cần cố thống nhất cách dịch giữa `lstm/gru cell` với `memory cell`, vì bản chất nó cũng chỉ kiểu đối tượng khác nhau (layer với array).

Reply:
Layer trong sequence này thường nhận đầu vào là 1 chuỗi, như mô tả ở stackoverflow thì gọi cell trong vòng for. Cũng có thể để đỡ rối thì họ đặt tên block bên trong là `LSTM cell`, còn LSTM layer đóng gói hết forward cho sequence. Nếu tách biệt LSTM cell này ra thì có lẽ `ô nhớ` ở đây phù hợp nhất.